src/app/results/results.component.ts

```diff
@@ -22,6 +23,9 @@ export class ResultsComponent implements OnInit {
   begin: number;
   message: string;
   query: any;
+  hoverBox: boolean = false;
```

Comment:
refer this link:- [https://github.com/palantir/tslint/issues/1653](https://github.com/palantir/tslint/issues/1653)